core/src/main/java/io/confluent/kafka/schemaregistry/rest/SchemaRegistryConfig.java

```diff
@@ -118,7 +118,7 @@
    * <code>mode.mutability</code>*
    */
   public static final String MODE_MUTABILITY = "mode.mutability";
-  public static final boolean DEFAULT_MODE_MUTABILITY = false;
+  public static final boolean DEFAULT_MODE_MUTABILITY = true;
```

Comment:
Is there any unit test to test this default?

Reply:
@maverick64 , added a test